modules/op-release-notes-1-1.adoc

```diff
@@ -0,0 +1,103 @@
+** Pipeline results are now added to the pipeline specification and `PipelineRun` status.
+** The `TaskRun.Status` field, now contains the exact Task specification used to instantiate the `TaskRun`.
+* Support to find the name of the PersistentVolumeClaim when a VolumeClaimTemplate is used as a volume source is now added using variable substitution.
+* Support to apply the default parameter to Conditions.
+* A TaskRun created by referencing a ClusterTask now adds the `tekton.dev/clusterTask` label instead of the `tekton.dev/task` label.
+* The `kubeconfigwriter` now adds the `ClientKeyData` and the `ClientCertificateData` configurations in the Resource structure to enable replacement of the pipeline resource type cluster with the kubeconfig-creator Task.
+* The feature-flags config map names and the defaults config map name are now customizable.
+* Support for HostNetwork in the PodTemplate used by TaskRun is now added.
+* An Affinity Assistant is now introduced to support node affinity in TaskRuns that share workspace volume. By default, this is disabled on OpenShift Pipelines.
+* The PodTemplate has been updated to specify `imagePullSecrets` to identify secrets that the container runtime should use to authorize container image pulls when starting a Pod.
+* Support for emitting warning events from the TaskRun controller if the controller fails to update the TaskRun.
+* Standard or recommended k8s labels are now added to all resources to identify resources belonging to an application or component.
+* The Entrypoint process is now notified for signals and they are propagated using a dedicated PID Group of the Entrypoint process.
+* The PodTemplate is now allowed to be set on a Task level at runtime using `TaskRunSpecs`
+* Support for emitting TaskRun lifecycle events such as the `taskrun started` and the `taskrun running` commands is added.
+* Support for using the `$context.<task|taskRun|pipeline|pipelineRun>.name` variable to reference the appropriate name when in PipelineRuns and TaskRuns.
+* The PipelineRun controller now emits a Kubernetes event every time that a Pipeline is started.
+CloudEvents can now be emitted for TaskRuns.
+* A new reason `PipelineRunStopping` is added to indicate that the PipelineRun has failed and is waiting for TaskRuns to complete, this ensures that the pipeline status is set when all Tasks complete.
+* Validation for PipelineRun parameters is now added to ensure that all the parameters required by the Pipeline are provided by the PipelineRun. This also allows PipelineRuns to provide extra parameters in addition to the required parameters.
+* Support for more cloud events is now added. The controller can be configured to send any TaskRun events, such as create, started, failed, as cloud events, in addition to the default Kubernetes events.
```

Comment:
@vdemeester isn't this the same as item on line 40? Could it be combined into one?